devfiles/osio-dogfooding.devfile.yaml

```diff
@@ -0,0 +1,101 @@
+# A devfile to develpop che-theia projects on che.openshift.io.
+# For developers who work on Theia core or che-theia core extensions and plugins.
+
+apiVersion: 1.0.0
+metadata:
+  name: theia-dogfooding
+projects:
+  - name: theia
+    source:
+      location: 'https://github.com/eclipse-theia/theia.git'
+      type: git
+      branch: master
+
+  - name: che-theia
+    source:
+      location: 'https://github.com/eclipse/che-theia.git'
+      type: git
+      branch: master
+
+components:
+  - id: eclipse/che-theia/latest
```

Comment:
hello, for dogfooding you prefer to use the latest than the next version ?

Reply:
I prefer to use latest stable release. But the developer build his latest che-theia from sources.